core/types.ml

```diff
@@ -136,11 +136,11 @@ type lens_name = string
 type lens_phrase =
   [ `Constant  of Constant.constant
   | `Var       of lens_name
-  | `InfixAppl of Operators.binop * lens_phrase * lens_phrase
-  | `UnaryAppl of Operators.unary_op * lens_phrase
+  | `InfixAppl of LensOperators.binop * lens_phrase * lens_phrase
```

Comment:
A good next step would be to invert this dependency too, such that `types -> lens extension`.